### ISSUE TITLE
TFS revscriptsys writing attacks which cause conditions is not working

### ISSUE THREAD

**Issue by soul4soul** [soul4soul/ot-monster-converter#8]
The logic for attacks which cause elemental damages is not working. Below is the only research that went into fixing the problem. This example was take from a Lich poison beam attack which causes poison condition.

This was initially used but fails to load.
```lua
{name ="poisoncondition", interval = 2000, chance = 10, length = 7, spread = 3, effect = CONST_ME_HITBYPOISON, target = false}
```

This is used now but doesn't make much sense since the condition to cause is not included. At least the fail to load problem went away.
```lua
{name ="condition", interval = 2000, chance = 10, length = 7, spread = 3, effect = CONST_ME_HITBYPOISON, target = false}
```

**Comment by soul4soul** [comment]
```
{name ="condition", type=CONDITION_POISON, interval = 2000, chance = 10, length = 7, spread = 3, effect = CONST_ME_HITBYPOISON, target = false}
```

**Comment by soul4soul** [comment]
Closed by #18
